pkg/maps/ctmap/gc/gc.go

```diff
@@ -60,6 +62,10 @@ func Enable(ipv4, ipv6 bool, restoredEndpoints []*endpoint.Endpoint, mgr Endpoin
 
 			if initialScan {
 				close(initialScanComplete)
+				if mapType != bpf.MapTypeLRUHash &&
+					option.Config.ConntrackGCProfile == option.ConntrackGCProfileLazy {
+					option.Config.ConntrackGCProfile = option.ConntrackGCProfileNormal
```

Comment:
Changing a variable from `option.Config` looks awkward doing it outside of the function that setup the options